src/Microsoft.Fast.Components.FluentUI/Components/FluentTextField.razor

```diff
@@ -9,12 +24,23 @@
 
     [Parameter] public bool? Autofocus { get; set; }
 
-    [Parameter] public Resize? Resize { get; set; }
+    [Parameter] public int? Size { get; set; }
 
     [Parameter] public Appearance? Appearance { get; set; }
 
+    [Parameter] public TextFieldType? TextFieldType { get; set; }
```

Comment:
I wonder if we should rename this to "Type" so that it matches the attribute names better. This one seems like it doesn't match the pattern of everything else right now.